packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.12.2
+
+* Fixes memory leak by implementing mechanism for disposing stream subscriptions in `GoogleMapController`. 
```

Comment:
Just this sentence is enough for the changelog; anyone interested in the level of detail in the following sentences can always look into the PR and/or issue.